src/test/java/unit/org/apache/tomcat/vault/util/EncryptionUtilTest.java

```diff
@@ -0,0 +1,161 @@
+package unit.org.apache.tomcat.vault.util;
+
+import org.apache.tomcat.vault.util.EncryptionUtil;
+import org.junit.Test;
+
+import javax.crypto.Cipher;
+import javax.crypto.KeyGenerator;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class EncryptionUtilTest {
+
+    private final String encryptionAlgorithm = "AES";
+    private final int keySize = 128;
+    private final EncryptionUtil encryptionUtil = new EncryptionUtil(encryptionAlgorithm, keySize);
+
+    @Test
+    public void testValidEncryption() throws Exception {
+        
+        String data = "Hello, World!";
+        byte[] inputData = data.getBytes();
+        SecretKey key = KeyGenerator.getInstance(encryptionAlgorithm).generateKey();
+
+        
+        byte[] encryptedData = encryptionUtil.encrypt(inputData, key);
+
+        
```

Comment:
I think we could merge the two empty lines into one. But if you find it clearer, I don't mind.